CMakeLists.txt

```diff
@@ -49,18 +51,15 @@ include_directories(
 )
 add_compile_definitions(AK_DONT_REPLACE_STD)
 
-foreach(prefix_dir IN LISTS CMAKE_PREFIX_PATH)
-    list(APPEND GI_TYPELIB_PATH "${prefix_dir}/lib64/girepository-1.0")
-endforeach()
-list(JOIN GI_TYPELIB_PATH ":" GI_TYPELIB_PATH_ENV)
+pkg_get_variable(GI_TYPELIB_PATH gobject-introspection-1.0 typelibdir)
 
 set(RESOURCE_FILES)
 function(add_blueprint source_file ui_file)
     add_custom_command(
         OUTPUT "${ui_file}"
         COMMAND "${CMAKE_COMMAND}"
         ARGS
-            -E env GI_TYPELIB_PATH="${GI_TYPELIB_PATH_ENV}"
+            -E env GI_TYPELIB_PATH="${GI_TYPELIB_PATH}"
```

Comment:
This breaks systems that export a custom `GI_TYPELIB_PATH` in the environment (NixOS), can we please keep that working?

Reply:
Can you check if doing this fixes your env?

```
pkg_get_variable(GI_TYPELIB_PATH gobject-introspection-1.0 typelibdir)
if ($ENV{GI_TYPELIB_PATH})
    list(APPEND GI_TYPELIB_PATH $ENV{GI_TYPELIB_PATH})
endif()
```

Reply:
oh wait that still needs the ; --> : conversion